files/image_config/sysctl/sysctl-net.conf

```diff
@@ -37,3 +37,4 @@ net.ipv4.udp_l3mdev_accept=1
 net.core.rmem_max=3145728
 net.core.wmem_max=3145728
 net.core.somaxconn=512
+net.ipv4.fib_multipath_use_neigh=1
```

Comment:
any equivalent for ipv6?

Reply:
@prsunny looks like for ipv6 it is handle differently as the default behavior https://github.com/torvalds/linux/blob/v4.19/net/ipv6/route.c#L611.

IPv6 does not even use CONFIG_IP_MULTIPATH option also.